compile.json

```diff
@@ -0,0 +1,52 @@
+{
+  "targets": [
+    {
+      "type": "build",
+      "outputPath": "lib",
+      "minify": "off",
+      "babelOptions": {
+        "targets": "node >= 10"
+      }
+    }
+  ],
+  "defaultTarget": "build",
+  "locales": [
+    "en"
+  ],
+  "libraries": [
+    "./framework"
+  ],
+  "applications": [
+    {
+      "class": "qx.application.Native",
```

Comment:
I wonder about the name "qx.application.Native" - wouldn't "qx.application.Server" be more fitting? What would "Native" mean in this context?

Reply:
The class here is needed by the compiler and is just an dummy which will be loaded during startup. 
The native application is the app for non GUI apps and do nothing.